docs/docs/hpc-on-k8s/minio.md

```diff
@@ -0,0 +1,34 @@
+# Deploying Minio in Kubernetes
+
+To deploy Minio in our Kubernetes cluster, we utilize Helm charts. Prior to installation, it's crucial to ensure that the Helm CLI is properly set up and the kubeconfig file is located in the designated directory.
+
+For this installation, we've opted for the default namespace, employing the following configurations:
+
+- storageClass:  
+- Minio rootUser
+- Minio rootPassword
+- Minio ServiceType
+- Minio API port
+- Minio console port
```

Comment:
I think it's worth doing something with this section, either:
- Explaining what these configurations are 
- Removing it entirely (as I don't see why we need to mention them here?) 

Regardless, using the CLI arg names (e.g. `rootUser` is what flags spellcheck errors however these can also be alleviated by wrapping them in backticks to make them in-line code). This'll also address removing these from codespell.

Reply:
updated thanks